projects/project-player-library/src/lib/pages/details-page/details-page.component.ts

```diff
@@ -71,7 +75,7 @@ export class DetailsPageComponent implements OnInit {
   taskCardAction(event:any){
     switch (event.action) {
       case 'edit':
-        this.moveToTaskDetails(event._id);
+        this.moveToDetailsTask(event._id);
```

Comment:
Why this file name change?

Reply:
there were two functions with same instant so removed the one function and used it